service/policy/db/migrations/20250401000000_add_actions_table.sql

```diff
@@ -0,0 +1,220 @@
+-- +goose Up
+-- +goose StatementBegin
+
+-- 1. Create new 'actions' table
+CREATE TABLE IF NOT EXISTS
+    actions (
+        id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
```

Comment:
on cascade delete?

Reply:
already handled below in id columns of `subject_mapping_actions`: https://github.com/opentdf/platform/pull/2040/files#diff-f0debee436c180c0ce384311e098293785a7e65c4c19a2d5aa01de718cc568a8R33-R34